admin-api/resource_server/api-platform.md

```diff
@@ -0,0 +1,64 @@
+---
+title: API Platform and CQRS integration
+menuTitle: API Platform and CQRS
+weight: 1
+---
+
+# API Platform and CQRS integration
+
+## CQRS based API
+
+Since the beginning of the Symfony migration, we chose CQRS as our base architecture, and their usage for the future API was already anticipated because:
+
+- it gives a more "domain approach" of our implementation, with more business logics than a simple CRUD
+- since the BO is already based on CQRS commands it guaranties a uniform behaviour between the BO and the API
+- all the CQRS commands and queries are behat tested, so they give a good insurance for stability
+- we capitalize on all the work that was done during the BO migration to build the Admin API
```

Comment:
```suggestion
- it gives a more "domain approach" to our implementation, with more business logic than a simple CRUD
- since the BO is already based on CQRS commands, it guarantees a uniform behaviour between the BO and the API
- all the CQRS commands and queries are Behat tested, so they give a good insurance for stability
- we capitalize on all the work that was done during the BO migration to build the Admin API
```